.github/workflows/deploy.yml

```
@@ -0,0 +1,102 @@
+
+      - name: "Deploy ActivityPub Queue to Cloud Run"
+        uses: 'google-github-actions/deploy-cloudrun@v2'
+        with:
+          image: europe-docker.pkg.dev/ghost-activitypub/activitypub/activitypub:${{ needs.build-test-push.outputs.activitypub_docker_version }}
+          region: ${{ matrix.region }}
+          service: stg-${{ matrix.region }}-activitypub-queue
+
+      - name: "Deploy ActivityPub API to Cloud Run"
+        uses: 'google-github-actions/deploy-cloudrun@v2'
+        with:
+          image: europe-docker.pkg.dev/ghost-activitypub/activitypub/activitypub:${{ needs.build-test-push.outputs.activitypub_docker_version }}
```

actionlint (1.7.4)

property "build-test-push" is not defined in object type {}

(expression)